### PR TITLE
lower threshold for using workdir on node

### PR DESCRIPTION
**Pull request by SalomeRonja**
fixes issue #49

### CODE REVIEW

**Comment by candidechamp** [comment]
@epbarros @SchroederB @SalomeRonja 

I haven't run many RE-EDS simulations lately so I haven't come across these issues but your fix seems good to me. 

I would suggest two things: 

- ask for the same number of s-values which determines where the calculation is run in the sopt and production submission scripts. 
- rename the local directory where the calculation will be run from '**scratch**' to '**localscratch**' (or something else) to avoid any confusion between the real scratch (on the node) and the directory we are creating. 

Finally, I know there have been quite a few changes on the Euler side where the reinstalled things so I'm surprised we still have this issue. Did anyone try contacting the Euler cluster support more recently with this problem?

File: reeds/function_libs/pipeline/worker_scripts/simulation_workers/RE_EDS_simulation_run_worker.py [hunk 1]
```diff
@@ -105,9 +105,9 @@ def work(out_dir: str, in_coord: str, in_imd_path: str, in_topo_path: str, in_pe
             os.system("mv " + work_dir + "/*  " + out_dir)
             # bash.move_file(in_file_path=work_dir+"/*", out_file_path=out_dir, verbose=True)
 
-        # post simulation cleanup
-        if not (isinstance(work_dir, type(None)) and work_dir == "None" and "TMPDIR" in os.environ):
-            bash.remove_folder(work_dir, verbose=True)
+        # post simulation cleanup -> for now, don't delete directory because it can lead to data loss during copying
```
Comment (SchroederB) [inline reeds/function_libs/pipeline/worker_scripts/simulation_workers/RE_EDS_simulation_run_worker.py:108]: this makes asense for the moment!

File: reeds/modules/do_RE_EDS_production.py [hunk 2]
```diff
@@ -164,7 +164,7 @@ def do(out_root_dir: str, in_simSystem: fM.System, in_template_imd: str,
 
 
         # fix for euler!
-        if (num_svals > 30):
+        if (num_svals > 20):
```
Comment (SchroederB) [inline reeds/modules/do_RE_EDS_production.py:167]: Isn't the general number 15 ? :)

**Review by SchroederB — APPROVED** [review]

**Comment by candidechamp** [comment]
@SchroederB 

My comment was about using the same number (e.g. 15) to determine the threshold at which we shift from running the calculation on scratch or locally, in all parts of the pipeline (sopt and production). This is similar to your code comment above.

**Comment by SchroederB** [comment]
> in
Ah I see! :)  yah that is a fair point (I also pointed it out ;) )

**Comment by SchroederB** [comment]
> @SchroederB
> 
> My comment was about using the same number (e.g. 15) to determine the threshold at which we shift from running the calculation on scratch or locally, in all parts of the pipeline (sopt and production). This is similar to your code comment above.

Ah I see! :) yah that is a fair point (I also pointed it out ;) )

**Review by epbarros — APPROVED** [review]
These changes look good to me, they solved the issue I was having. Thanks, @SalomeRonja!

I agree with the other comments that it would be good to keep the number of s values consistent.

**Comment by SalomeRonja** [comment]
Hey everyone, thanks a lot for the feedback, I implemented your suggestions, so since everyone seems to be happy with the changes, I'll do the merge now.

**Pull request merged**